### PR TITLE
mgmt compute, update api-version to 2022-08-01

### PR DESCRIPTION
**Pull request by XiaofeiCao**
# Description

Fix https://github.com/Azure/sdk-release-request/issues/3205.
No manual code, live tests pass.

Breaking change discussed and approved in https://github.com/Azure/azure-rest-api-specs/pull/20097#discussion_r941751308.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by XiaofeiCao** [comment]
Public IP addresses with basic skus can no longer be created with zone resiliency. https://github.com/Azure/sdk-release-request/issues/3205#issuecomment-1254413032

Will wait for service justification.

**Comment by XiaofeiCao** [comment]
It's a behavioral change from Network RP. I'll investigate whether it's introduced by new version.

For now, I think we can change the implicitly created pip sku to `standard` if zone resiliency is enabled on vm.

**Review by weidongxu-microsoft — APPROVED** [review]

File: sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,9 @@
...
 ## 2.19.0-beta.1 (Unreleased)
 
-### Features Added
-
 ### Breaking Changes
 
-### Bugs Fixed
-
-### Other Changes
+- Since `2021-05-01`, `PublicIpAddress` of `Basic` Sku no longer supported configuring with Availability Zone.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md:7]: please also give our version 2.1x

**Review by haolingdong-msft — APPROVED** [review]
LGTM. May need to add back `add-opens` to make LIVE and RECORD tests pass.

File: sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md (hunk 1 above)
Comment (XiaofeiCao) [reply sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md]: Added.

**Comment by XiaofeiCao** [comment]
> LGTM. May need to add back `add-opens` to make LIVE and RECORD tests pass.

Added.

**Pull request merged**